### PR TITLE
Cmm readme

### PR DESCRIPTION
**Pull request by chmorgan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Note that packet dissection and creation was split from SharpPcap some years ago
   * Live device lists
   * Statistics
   * Reading packets from Live Devices (actual network devices) and Offline Devices (Capture files)
-  * Support for Berkley Packet Filters
+  * Support for [Berkley Packet Filters](https://www.tcpdump.org/manpages/pcap-filter.7.html)
```
Comment (kayoub5) [inline README.md:28]: Please keep in mind that WinDivert uses a different syntax for filtering

**Review by kayoub5 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chmorgan) [reply README.md:28]: Should we note that?

Comment (kayoub5) [reply README.md:28]: In `WinDivertDevice.Filter` at least.

**Comment by chmorgan** [comment]
@kayoub5 added a link to the WinDivert docs for filtering. This seems like a good approach. Not sure it makes sense for use o do any kind of pre-processing on the filter input. Thoughts on whether we are doing what we can/should do here around filtering?

**Pull request merged**